Services/CO.CDP.Forms.WebApi/UseCase/UpdateFormSectionAnswersUseCase.cs

```diff
@@ -23,9 +23,51 @@ public async Task<bool> Execute((Guid formId, Guid sectionId, Guid answerSetId,
         var section = await formRepository.GetSectionAsync(formId, sectionId)
             ?? throw new UnknownSectionException($"Unknown section {sectionId} in form {formId}.");
 
-        var sharedConsent = await formRepository.GetSharedConsentDraftWithAnswersAsync(formId, organisationId)
-                            ?? CreateSharedConsent(organisation, section.Form);
+        var sharedConsent = await formRepository.GetSharedConsentWithAnswersAsync(formId, organisationId);
+        if (sharedConsent == null)
+        {
+            sharedConsent = CreateSharedConsent(organisation, section.Form);
+        }
+        else if (sharedConsent.SubmissionState == Persistence.SubmissionState.Submitted)
```

Comment:
Add unit test for copying existing Answer set excluding Declaration section scenario.

Reply:
added